### ISSUE TITLE
In solarized_dark theme, the errors and warnings look very similar.

### ISSUE THREAD

**Issue by adsnaider** [helix-editor/helix#2336]
### Summary

The color of the warning symbols (with LSP) in the solarized_dark theme look very similar to the error symbols. Personally, I think the warnings are too reddish (See the image below).

![Screenshot from 2022-04-29 21-47-23](https://user-images.githubusercontent.com/11418516/166091517-cdd594d9-18b8-4522-a0c6-f79f0b2c6748.png)

The top is a warning and the bottom an error.

### Reproduction Steps

 

I tried this:

1. `hx foo.rs`
2. :theme solarized_dark
3. Cause a rust_analyzer warning and rust_analyzer error

### Helix log

N/A

### Platform

Linux

### Terminal Emulator

gnome-terminal

### Helix Version

22.03-137-g8eb15f52

**Comment by the-mikedavis** [comment]
Yeah the orange and red colors look too close. Maybe using the pinkish color like in `#[test]` would be an improvement?

**Comment by colaxdev** [comment]
Just modify the theme file
